### ISSUE TITLE
[SF][a11y] ui5-multi-input: Input should be focused when all the tokens are deleted

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#6073]
### **Bug Description**
When all the selected tokens in `ui5-multi-input` are deleted, the focus is lost.

### **Expected Behavior**
The input itself should be focused when all the tokens are deleted.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/MultiInput/
2. Go to `Multi Input and token creation onChange` section
3. Type `a` in the input and select `Argentina` item
4. Delete `Argentina` token and the focus is lost

### **Context**
- UI5 Web Components version: 1.9.1
- OS/Platform: MacOS
- Browser: Edge
- Affected component: ui5-multi-input

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

When we have <ui5-multi-input> with tokens that could be deleted, deleting the last token the focus is not moved to the input field but lost (goes to the body).

The behaviour is reproducible both using keyboard and deleting a token via its clear icon.
On the other hand, if we delete a token but there are more tokens remaining in the ui5-multi-input, the focus is moved to the previous token, not to the body, so in that case it works as expected.

Could you please look over the issue?

Thank you in advance!

Kind Regards,
Niya
